yt/visualization/plot_modifications.py

```diff
@@ -1932,18 +1932,20 @@ def __init__(
         self.plot_args = {
             "color": "white",
             "s": 50,
-            "transform": None,
             **(plot_args or {}),
             **kwargs,
         }
         self.coord_system = coord_system
+        self.transform = None
 
     def __call__(self, plot):
         x, y = self._sanitize_coord_system(
             plot, self.pos, coord_system=self.coord_system
         )
         x, y = self._sanitize_xy_order(plot, x, y)
-        plot._axes.scatter(x, y, marker=self.marker, **self.plot_args)
+        plot._axes.scatter(
+            x, y, marker=self.marker, transform=self.transform, **self.plot_args
+        )
```

Comment:
Will this always be `None` or are there cases where it will not be?  Is that because of the `coord_system` usage?

Reply:
`_sanitize_coord_system` is allowed to modify `self.transform`, this is what I missed in my breaking PR